[ML] Adding a known issue for the ML datafeed auth header problem (#92274)
elastic/elasticsearch: docs/reference/release-notes/8.4.0.asciidoc

Also see <<breaking-changes-8.4,Breaking changes in 8.4>>.

[[known-issues-8.4.0]]
[float]
=== Known issues

// tag::ml-pre-7-datafeeds-known-issue[]

> * {ml-cap} {dfeeds} cannot be listed if any are untouched since version 6.x

Reviewer: ```suggestion
* {ml-cap} {dfeeds} cannot be listed if any are not modified since version 6.x
```

+
If you have a {dfeed} that was created in version 5.x or 6.x and has not

> been updated since 7.0 it is not possible to list {dfeeds} in versions

Reviewer: ```suggestion
been updated since 7.0, it is not possible to list {dfeeds} in 
```

> 8.4 and 8.5. This issue means that {anomaly-jobs} cannot be managed using

Reviewer: ```suggestion
8.4 and 8.5. This means that {anomaly-jobs} cannot be managed using
```

> {kib}. This problem is fixed in version 8.6.

Reviewer: ```suggestion
{kib}. This issue is fixed in 8.6.
```

+

> If you upgrade to version 8.4 or 8.5 with such a {dfeed} you will need to

Reviewer: ```suggestion
If you upgrade to 8.4 or 8.5 with such a {dfeed}, you need to
```
